summarize/src/analysis.rs

```diff
@@ -254,7 +254,9 @@ pub fn perform_analysis(data: ProfilingData) -> Results {
                 thread.stack.push(current_event)
             }
             EventPayload::Integer(value) => {
-                artifact_sizes.push(ArtifactSize::new(current_event.label.into_owned(), value))
+                if current_event.event_kind == measureme::rustc::ARTIFACT_SIZE_EVENT_KIND {
```

Comment:
We should be able to use `ARTIFACT_SIZE_EVENT_KIND` without the qualification since we already import `measureme::rustc::*` at the top of this file.